settings.xml

```diff
@@ -1,54 +1,62 @@
 <!--
-
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
-
+    Copyright (c) 2018, 2020 Oracle and/or its affiliates. All rights reserved.
     This program and the accompanying materials are made available under the
     terms of the Eclipse Public License v. 2.0, which is available at
     http://www.eclipse.org/legal/epl-2.0.
-
     This Source Code may also be made available under the following Secondary
     Licenses when the conditions for such availability set forth in the
     Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
     version 2 with the GNU Classpath Exception, which is available at
     https://www.gnu.org/software/classpath/license.html.
-
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
-
 -->
 
 <settings>
+      <localRepository>${env.PORTING}/.m2/repository</localRepository>
```

Comment:
This is going to slow things down. This dir on CI will be created new each time, so all dependencies will be needed to be downloaded for each module!